plugins/modules/notification/slack.py

```diff
@@ -68,7 +68,8 @@
     type: str
   message_id:
     description:
-      - Optional. Message ID to edit, instead of posting a new message.
+      - Optional. Message ID to edit, instead of posting a new message. 
```

Comment:
Here's the trailling space:
```suggestion
      - Optional. Message ID to edit, instead of posting a new message.
```